public/app/features/dashboard/state/PanelQueryRunner.ts

```diff
@@ -238,8 +204,9 @@ export class PanelQueryRunner {
       this.subject.complete();
```

Comment:
As mentioned in my comment in `subscribe` method I do not think this will actually notify anybody.

Reply:
true, currently none of the subscribers to PanelQueryRunner care about completions but could be an issue when I add shared query feature between panels, then it could matter.